doc/website/mkdocs.yml

```diff
@@ -0,0 +1,118 @@
+site_name: Holoscan Reference Applications
+site_url: https://nvidia-holoscan.github.io/holohub
+repo_url: https://github.com/nvidia-holoscan/holohub
+repo_name: nvidia-holoscan/holohub
+
+theme:
+  name: "material"
+  custom_dir: overrides
+  features:
+    - content.code.annotate
+    - navigation.tabs
+    - navigation.indexes
+    - navigation.instant
+    - navigation.path
+    - navigation.instant.prefetch
+    - navigation.top
+    - navigation.footer
+    - search.suggest
+    - search.highlight
+    - content.code.copy
+  font:
+    code: Roboto Mono
+  favicon: assets/images/favicon.png
+  language: en
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: light
+      primary: custom
+      accent: lime
+      toggle:
+        icon: material/weather-night
+        name: Switch to dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: dark
+      primary: custom
+      accent: lime
+      toggle:
+        icon: material/weather-sunny
+        name: Switch to light mode
+  icon:
+    logo: nvidia/nvidia-logo
+    repo: fontawesome/brands/github
+
+extra_css:
+  - assets/css/color-schemes.css
+  - assets/css/fonts.css
+  - assets/css/custom-material.css
+  - assets/css/jupyter-themes.css
+
+plugins:
+  - search
+#  - git-revision-date-localized:
+#      fallback_to_build_date: true
```

Comment:
Remove?

Reply:
I'd like to keep it in case we want to use it.